setup-container/setup/install_kubernetes.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+node=$1
+if [ "$node" != "master" ] && [ "$node" != "minion" ]; then
+  "You must specify 'master' or 'minion' as argument"
+  exit 1
+fi
+
+# copy hyperkube binary
+cp /hyperkube /hostfs/usr/bin/
+
+# copy kubelet service configuration
+cp /setup/kubelet.service /etc/systemd/system/
+
+if [ "$node" == "master" ]; then
+  # clean up old configuration
+  rm -rf /hostfs/var/lib/kubelet
```

Comment:
I think we do this in the wrapper script. I assume this is the better place for cleanup